string program_name
---
# Success if empty
string error
string native_code
